src/main/java/com/bbaga/githubscheduledreminderapp/domain/notifications/slack/ChannelMessageBuilder.java

```diff
@@ -142,16 +144,16 @@ public SectionBlock createLine(GitHubIssue issue, String text, Map<String, Strin
             login = "Unknown";
         }
 
-        var processedText = text.replaceAll("\\$login", login)
-                .replaceAll("\\$title", issue.getTitle())
-                .replaceAll("\\$repository", issue.getRepository().getFullName())
-                .replaceAll("\\$age", getIssueAge(issue))
-                .replaceAll("\\$assignee-logins", getAssigneeLogins(issue))
-                .replaceAll("\\$assignee-login-links", getAssigneeLoginLinks(issue))
-                .replaceAll("\\$link", getUrl(issue, trackingParams));
+        var processedText = text.replace("\\$login", login)
+                .replace("\\$title", issue.getTitle())
+                .replace("\\$repository", issue.getRepository().getFullName())
+                .replace("\\$age", getIssueAge(issue))
+                .replace("\\$assignee-logins", getAssigneeLogins(issue))
+                .replace("\\$assignee-login-links", getAssigneeLoginLinks(issue))
+                .replace("\\$link", getUrl(issue, trackingParams));
```

Comment:
When running the tests against Java 17, this line [produces an exception](https://github.com/bbaga/github-scheduled-reminder-app/actions/runs/3516561616/jobs/5895516111#step:4:74). The issue seems to be the `replaceAll` tolerated `null` as the second argument and `replace` does not.

Seems like [the `from` method](https://github.com/bbaga/github-scheduled-reminder-app/pull/222/files#diff-2b91b7fb225ac09846d25efa58f18de69c43fdf6d97a63052739b3e74bbf6d66R182) of [the urlBuilder mock is not prepared properly](https://github.com/bbaga/github-scheduled-reminder-app/pull/222/files#diff-9b10e4689b244dc3cd2946618a3eba4027b8a7a475c23698bb7bd513bdf1726bR66).

Reply:
My fault for trusting Sonarcube here....